randovania/games/planets_zebeth/layout/planets_zebeth_cosmetic_patches.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+import dataclasses
+from enum import Enum
+
+from randovania.games.game import RandovaniaGame
+from randovania.layout.base.cosmetic_patches import BaseCosmeticPatches
+from randovania.lib import enum_lib
+
+
+class PlanetsZebethRoomGuiType(Enum):
```

Comment:
with it being used 3 times now, i wonder whether we should have an issue that makes this a general class....